_pytest/config.py

```diff
@@ -991,7 +991,8 @@ def pytest_load_initial_conftests(self, early_config):
 
     def _initini(self, args):
         ns, unknown_args = self._parser.parse_known_and_unknown_args(args, namespace=self.option.copy())
-        r = determine_setup(ns.inifilename, ns.file_or_dir + unknown_args, warnfunc=self.warn)
+        rootdir = ns.rootdir if ns.rootdir else None
+        r = determine_setup(ns.inifilename, ns.file_or_dir + unknown_args, warnfunc=self.warn, rootdir_cmd_arg=rootdir)
```

Comment:
This could be written as:

```python
r = determine_setup(ns.inifilename, ns.file_or_dir + unknown_args, warnfunc=self.warn, rootdir_cmd_arg=ns.rootdir or None)
```

Besides shorter, this doesn't left a `rootdir` local variable to be used by the unaware. 😁